templates/new_gem/lib/fluent/plugin/storage.rb.erb

```diff
@@ -0,0 +1,50 @@
+<%= preamble %>
+
+require "fluent/plugin/storage"
+
+module Fluent
+  module Plugin
+    class <%= class_name %> < Fluent::Plugin::Storage
+      Fluent::Plugin.register_storage("<%= plugin_name %>", self)
+
+      attr_reader :store
+
+      # Set persistent true by default
+      config_set_default :persistent, true
+
+      def initialize
+        super
+        @store = {}
+      end
+
+      def configure(conf)
+        super
+      end
+
+      def multi_workers_ready?
```

Comment:
It's not needed, because same implementation exists in `Fluent::Plugin::Base`

Reply:
Oh, it is not needed.